packages/editor/src/components/post-card-panel/index.js

```diff
@@ -106,3 +130,59 @@ export default function PostCardPanel( { className, actions } ) {
 		</PanelBody>
 	);
 }
+
+// Taken from packages/editor/src/components/time-to-read/index.js.
+const AVERAGE_READING_RATE = 189;
+
+// This component renders the wordcount and reading time for the post.
+function PostContentInfo( { postContent } ) {
+	/*
+	 * translators: If your word count is based on single characters (e.g. East Asian characters),
+	 * enter 'characters_excluding_spaces' or 'characters_including_spaces'. Otherwise, enter 'words'.
+	 * Do not translate into your own language.
+	 */
+	const wordCountType = _x( 'words', 'Word count type. Do not translate!' );
+	const wordsCounted = postContent
+		? wordCount( postContent, wordCountType )
+		: 0;
+	if ( ! wordsCounted ) {
+		return null;
+	}
+	const readingTime = Math.round( wordsCounted / AVERAGE_READING_RATE );
+	let wordsCountText;
+	if ( ! wordsCounted.toLocaleString() ) {
+		wordsCountText = __( 'Unknown' );
+	} else {
+		wordsCountText =
+			wordsCounted === 1
+				? __( '1 word' )
+				: sprintf(
+						// translators: %s: the number of words in the post.
+						_n( '%s word', '%s words', wordsCounted ),
+						wordsCounted.toLocaleString()
+				  );
+	}
+	const readingTimeText =
+		readingTime <= 1
+			? __( '1 minute read time' )
+			: sprintf(
+					// translators: %s: the number of words in the post.
+					_n(
+						'%s minute read time',
+						'%s minutes read time',
+						readingTime
+					),
+					readingTime.toLocaleString()
+			  );
+	return (
+		<Text>
+			{ sprintf(
+				/* translators: 1: How many words a post has(eg. 30 words). 2: the number of minutes of read time(eg. 2 minutes read time) */
+				/* translators: %1s: is the number of minutes. */
```

Comment:
* You're missing the second parameter.
* I think this should come with a full example — "400 words, 2 minutes reading time"

Reply:
Oh.. the second line was leftover, but I had the comment for the second param. I'll update though with a full example.